Microsoft.NET.Build.Containers/Image.cs

```diff
@@ -54,7 +54,10 @@ public void AddLayer(Layer l)
         RecalculateDigest();
     }
 
-    private void RecalculateDigest() {
+    private void RecalculateDigest()
+    {
+        config["created"] = DateTime.UtcNow;
```

Comment:
what datetime format is expected for this config entry?

Reply:
A fine question! The OCI spec says [RFC 3339, section 5.6](https://tools.ietf.org/html/rfc3339#section-5.6) which points to ISO 8601 which is the STJ default per https://docs.microsoft.com/dotnet/standard/datetime/system-text-json-support

Reply:
And Docker Desktop seems happy enough with this:

![image](https://user-images.githubusercontent.com/3347530/183771606-a2adbf78-d476-4301-b5ec-18b7fb400c4d.png)

```sh-session
❯ docker image list
REPOSITORY                                  TAG       IMAGE ID       CREATED          SIZE
dotnetcontainers/testimage                  latest    6cea448eec7a   24 minutes ago   260MB
localhost:5010/dotnet/runtime               6.0       a90b8fb52209   8 hours ago      188MB
mcr.microsoft.com/dotnet/runtime            6.0       a90b8fb52209   8 hours ago      188MB
```